app/models/reply.rb

```diff
@@ -97,7 +97,7 @@ def notify_other_authors
   end
 
   def previous_reply
-    @prev ||= post.replies.find_by(reply_order: reply_order - 1)
+    @prev ||= post.replies.find_by(reply_order: reply_order - 1) || post.written
```

Comment:
I'm not 100% sure we want post.written here, but it seems like we only use it in `reply#delete` (and it'd give 1 here, and defaults to 1), so it's probably okay (right?).

Reply:
I think the reason I did that is so that `set_previous_reply_to_last` doesn't wind up with nil. Possibly it should? Idk.